tonic/models/vic/grid_params.py

```diff
@@ -602,6 +732,71 @@ def __init__(self, organic_fract=False, spatial_frost=False,
                                'basin_depth': 'm',
                                'basin_area': 'fraction'}
 
+        # State Variables
+        self.state = {'gridcell': 'N/A',
+                      'Soil_moisture': 'mm',
+                      'Soil_ice': 'mm',
+                      'Canopy_water': 'mm',
+                      'Snow_age': 'time steps',
```

Comment:
Do we store the simulation timestep somewhere?  If not, this variable should probably be in different units (e.g. seconds) and we should open an issue on github.